### ISSUE TITLE
scim not working properly in openbox 3.6.1

### ISSUE THREAD

**Issue by redf0x** [scim-im/scim#23]
I've run into problem using scim under openbox. [Here](https://forums.gentoo.org/viewtopic-t-1073114.html) are some details and efforts to investigate what might cause the problem. Today I upgraded to scim 1.4.17 - still the same. Please, let me know if you need further input on this issue as I have no clue what else might affect scim besides DE and cause this odd behaviour.
TA

**Comment by tzhuan** [comment]
Thanks for your bug report. A few questions:
 - Do you use gcc5 or gcc6 to build scim 1.4.17?
 - Could you please try to remove scim-anthy and scim-pinyin to see if the scim works with only scim-tables?

**Comment by redf0x** [comment]
I used gcc6 to build scim 1.4.17. However before update to scim 1.4.17 I had scim 1.4.14 which failed to compile with gcc6, so I used gcc5 to build scim 1.4.14 itself and its ilk, while the rest of the system was built with gcc6.
I'm sorry about confusion. It seems, scim 1.4.17 works well in openbox and the problem is gone. For some reason I forgot to re-set the environment variables and hence scim 1.4.17 was simply not in use by the clients. ATM everything looks ok with scim 1.4.17. And it works with only tables as well as with anthy and pinyin. I think this issue may be closed for now.
